api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -91,6 +91,14 @@ message RedisProxy {
     // downstream unchanged. This limit defaults to 100.
     google.protobuf.UInt32Value max_upstream_unknown_connections = 6;
 
+    // Enable per-command statistics per upstream cluster, in addition to the filter level aggregate
+    // count.
+    bool enable_command_stats = 8;
+
+    // Indicates that latency stat should be computed in microseconds. By default it is computed in
+    // milliseconds.
+    bool latency_in_micros = 9;
+
```

Comment:
Discussed with Henry offline, we will only record Redis latency in micros (millis is useless for this).